exporter/terra/submission/responder.py

```diff
@@ -27,16 +27,18 @@ def __init__(self, ingest_service: IngestService, gcp_project: str, gcp_topic: s
             except AlreadyExists:
                 self.logger.info(f'Subscription Found: {self.subscription_path}')
             except Exception as e:
-                self.logger.info(f'Cannot check whether subscription exists: {self.subscription_path} due to {e}')
+                self.logger.info(f'Cannot check whether subscription exists: {self.subscription_path} due to {str(e) if str(e) else e.__class__.__name__}')
```

Comment:
Is this an info or a warning?

Reply:
info, it is an expected state now that Nate he de-elevated our permissions.